ChangeLog.md

```diff
@@ -1,34 +1,24 @@
 # deblur changelog
 
-## Version 0.1.9-dev
-### Features
-Updated docs
-Delete tmp files and split directory in deblur output directory
-
-### Backward-incompatible changes [stable]
-Remove the --skip-trimming flag and use --trim-length=-1 instead
-Remove the --negate flag, always output results of only negative, hits to positive database and non-hits to positive database
-Change output file names to all.biom, all.seqs.fa, reference-non-hit.biom, reference-non-hit.seqs.fa, reference-hit.biom, reference-hit.seqs.fa
-
-### Performance enhancements
-
-### Bug fixes
+## Version 1.0
 
+Official version 1.0. Released on 9 February 2017.
 
-## Version 0.1.8-dev
 ### Features
-Updated docs
+* Updated documentation to improve clarity on examples, file inputs and file outputs.
+* Delete tmp files and split directory in deblur output directory
 
 ### Backward-incompatible changes [stable]
-Change --min-reads default value to 10 (was 0)
-Remove --trim-length default value (was 100) and set as required parameter
-Set --negate (negative mode) default value to True and add --only-pos option to skip negative mode
+* Removed the --skip-trimming flag and use --trim-length=-1 instead
+* The trim length parameter is now required.
+* Set --negate (negative mode) default value to True; specify --only-pos option to skip negative mode. What this means is, by default, any read which appears to be PhiX or Adapter will be filtered out.
```

Comment:
we completely removed the --negate flag. So no whenever you run deblur, it automatically does the negative filtering and positive filtering.